assets/global.js

```diff
@@ -422,12 +428,12 @@ class DeferredMedia extends HTMLElement {
   }
 
   loadContent() {
+    window.pauseAllMedia();
```

Comment:
@ludoboludo moving this out of the loaded conditional below seems to fix the issue like you suggested. Let me know if this isn't what you had in mind.

Reply:
Nope that's what I was thinking too 👌